### DISCUSSION TITLE
Anime Torrent Fansub

### DISCUSSION THREAD

**Discussion by MainKronos** [MainKronos/Sonarr-AnimeDownloader#17] [General]
### Una lista di Funsub che rilasciano anime tramite Torrent:

- Akindo = Akindo Fansub (https://akindofansub.forumfree.it/)
- Akuma = Akuma (???)
- AOneITA = Anime One ITALIA (http://aoneita.altervista.org/)
- Aozora = aozer0 (http://www.aozer0.com/)
- BBF = Bowling Ball Fansubs (http://bowlingballfansubs.it/)
- BFE = Best Fansub Ever (https://bestfansubever.forumfree.it)
- BKT = BinKanTeam (https://narutoninja.forumfree.it/)
- BMzone = BM zone (https://bmzone.forumfree.it/)
- Demone Celeste = Demone Celeste (https://www.demoneceleste.it/)
- DN_Team = D.N. Team Fansub (http://dnfansubanime.blogspot.com/)
- Emblem-Raws = Emblem Subs (https://www.emblem-subs.org/)
- Emblem-Subs = Emblem Subs (https://www.emblem-subs.org/)
- Emblem = Emblem Subs (https://www.emblem-subs.org/)
- EmblematicheTette = Emblem Subs (https://www.emblem-subs.org/)
- F&Fs = Funny and Fantasy Subs (https://fefs.it/)
- Fate = Fate (???)
- Fnanna Fansub = Fnanna Fansub (https://wlafnanna.blogfree.net/)
- Freedom = Freedom (https://animecuriosita.forumfree.it)
- GTO = GTO THE GREAT SITE (https://www.gtothegreatsite.net/)
- Hanasaku = Hanasaku Subs (https://www.hanasaku-subs.com/)
- Instantia Fansub = Instantia Subs (https://instantiafansub.wordpress.com/)
- Jimaku = Jimaku-Subs (http://jimakusubs.altervista.org/)
- Kanji = KanjiSubs (http://www.kanjisub.com/)
- Kanjisub = KanjiSubs (http://www.kanjisub.com/)
- KF = KUMACHI FANSUB (http://kumachifansub.altervista.org/)
- Kumachi = KUMACHI FANSUB (http://kumachifansub.altervista.org/)
- LEAF = LEAF FANSUB (http://leaf.lacumpa.biz/)
- Lovely = lovelysubs (https://lovelysubs.weebly.com/)
- LovelyCr = lovelysubs (https://lovelysubs.weebly.com/)
- LovelyFire = lovelysubs (https://lovelysubs.weebly.com/)
- LovelySubs = lovelysubs (https://lovelysubs.weebly.com/)
- LS = lovelysubs (https://lovelysubs.weebly.com/)
- MAS = MANGA ANIME SEKAI (http://mangaanimesekai.netsons.org/)
- Nippart = Nippart Manga & Anime Orange (https://nippart.forumfree.it/)
- Omnivium = Omnivium (https://omniviumfansub.github.io/)
- OnF-Supremes = OnF-Supremes (http://www.supremes.it/)
- OPF = ONE PIECE FANSUB (http://www.opfitalia.net/index.php?start=5)
- OPIF = One Piece Italian Funsub (https://itaonepiece.forumfree.it/)
- Otacosub = OtacoSub (https://msubweb.wordpress.com/)
- Owari = OWARI SUBS GO AHEAD (http://owarisubs.lacumpa.biz/)
- Owari Sub = OWARI SUBS GO AHEAD (http://owarisubs.lacumpa.biz/)
- Pantaloni Rossi = Pantaloni Rossi (https://pantalonirossi.lacumpa.biz/)
- Pluschan = Pluschan Fansub (http://pluschan.lacumpa.biz/)
- Pluschan Fansub = Pluschan Fansub (http://pluschan.lacumpa.biz/)
- Pluschan_Fansub = Pluschan Fansub (http://pluschan.lacumpa.biz/)
- Rakuen = Rakuen Subs (https://www.rakuen-subs.net/)
- RT = Revolution Team (https://revolutionteam.forumcommunity.net/)
- RTeam = Revolution Team (https://revolutionteam.forumcommunity.net/)
- Ryusan = Ryusan-Fansub (http://ryusanfansub.altervista.org/)
- Saimoe = Saimoe Fansub (http://saimoefansub.blogspot.com/)
- SSK = SHINSEI-KAI (http://www.shinsei-kai.biz/)
- Supremes = Supremes (???)
- TAFF = TUTTOANIMEFF (https://tuttoanimeff.home.blog/)
- Task-Force = Task-Force (https://bmzone.forumfree.it/?f=64802816)
- Task Force = Task Force (???)
- TF = Tonkatsu Fansub (https://tonkatsufansub.wordpress.com/)
- TnS = Tanoshimi no Sekai (https://tnsfansub.com/)
- Tonkatsu = Tonkatsu Fansub (https://tonkatsufansub.wordpress.com/)
- UWS = Under World Subs (http://underworldsubs.org/)
- Whine-Subs = WHINESUBS (https://whinesubs.wordpress.com/)
- Whine = WHINESUBS (https://whinesubs.wordpress.com/)
- WhineSubs = WHINESUBS (https://whinesubs.wordpress.com/)
- XIII's Fansub = XIII's Fansub (https://xiiifansub.wordpress.com/)
- XIII = XIII's Fansub (https://xiiifansub.wordpress.com/)
- Yaoi801 = YAOI801 FANSUB (http://www.yaoi801fansub.it/)

La lista è formattata in questo modo:
`nome del funsub che si trova nel file torrent` = `Nome Funsub` (`link`)

---

Spero che la lista vi posso tornare utile come lo è stato per me👍
Se ne conoscete altri scrivete qui sotto.

**Comment by phoenix872** [comment]
i like anime
